### DISCUSSION TITLE
Working with JWT / newbe question.

### DISCUSSION THREAD

**Discussion by leostereo** [quasarframework/quasar#9014] [General - Components / Directives / etc]
Hi guys , I need to prepare my self to work on a middle complexity platform.
I will need to create register and login functions and after that handling the JWT provided by be api.
I do understand the basics of jwt authentication , however I never wrote code for this.
I would like to encourage this with bests practices possible. 
Is there some nice documentation / tutorial about this issue ?
Is there some module better suited for login / authentication handling ?
Thank you all for you feedback on this.
Leandro.

**Comment by ontwikkelfabriek** [comment]
You could look at these:
https://github.com/stefanvanherwijnen/quasar-app-extension-http-authentication
https://github.com/neatpro/Quasar-JWT
https://forum.quasar-framework.org/topic/5100/how-can-i-pass-the-jwt-token-to-axios-get-request/6
https://jasonwatmore.com/post/2018/07/06/vue-vuex-jwt-authentication-tutorial-example
https://dev.to/rachel_cheuk/part-1-user-roles-and-management-quasar-8jp

**Comment by corepay** [comment]
HI just ran across this...I've been using FeathersJS for my Auth and JWT and Feathers-Vuex.  The server side takes maybe a minute to get a working user authentication and feathers vuex is hand in glove.

**Comment by ap1969** [comment]
Hi Leandro,
Not sure if this it the way you want to go, but I used auth0 for my authentication. The Vue examples they provide should get you started, and it's working in SPA, PWA, and Cordova modes for me.

Andy

**Reply by leostereo** [reply]
thanks !!! 
I already begin with cognito from aws , there is an amplify cli  that provides all you need to begin.
